tests/datasets.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import dataclass
+
+import numpy as np
+import pandas as pd
+from sklearn.model_selection import train_test_split
+
+from sknnr.datasets import load_moscow_stjoes
+
+
+@dataclass
+class TestDataset:
```

Comment:
This may not be important and feel free to push back, but perhaps a bit more specificity on class naming here?  Perhaps just to signify that this test data is associated with NN modeling.  I think even `TestKNNDataset` would be fine.

Reply:
Sure, I like that. Actually, if we rename it `KNNTestDataset` we can avoid the `__test__ = False` that I had to put in because `pytest` thinks any class starting with `Test` is a unit test, so win-win!